python/cuda_parallel/tests/test_radix_sort.py

```diff
@@ -12,6 +12,13 @@
 
 import cuda.parallel.experimental.algorithms as algorithms
 
+
+def get_mark(dt, log_size):
+    if log_size < 20:
+        return tuple()
```

Comment:
It is odd that `marks` cannot be `None` here...

Reply:
@leofang `None` was my first attempt.

```
In [13]: import pytest

In [14]: pytest.param.__annotations__
Out[14]: 
{'values': 'object',
 'marks': 'MarkDecorator | Collection[MarkDecorator | Mark]',
 'id': 'str | None',
 'return': 'ParameterSet'}
```

The default value for `marks` keyword argument was `tuple()`, so this is what replacing `None` with worked.